### ISSUE TITLE
Cpptraj interactive mode can crash out with a long command line

### ISSUE THREAD

**Issue by drroe** [Amber-MD/cpptraj#958]
Problem is the `CpptrajFile::Printf` buffer gets overrun. Need to either fix `Printf` or use `Write` instead.

**Comment by drroe** [comment]
This should work:
```diff
diff --git a/AmberTools/src/cpptraj/src/Cpptraj.cpp b/AmberTools/src/cpptraj/src/Cpptraj.cpp
index f6857fab2f..5d94b8dacb 100644
--- a/AmberTools/src/cpptraj/src/Cpptraj.cpp
+++ b/AmberTools/src/cpptraj/src/Cpptraj.cpp
@@ -650,8 +650,11 @@ int Cpptraj::Interactive() {
   if (logfile_.IsOpen()) {
     // Write logfile header entry: date, cmd line opts, topologies
     logfile_.Printf("# %s\n", TimeString().c_str());
-    if (!commandLine_.empty())
-      logfile_.Printf("# Args: %s\n", commandLine_.c_str());
+    if (!commandLine_.empty()) {
+      logfile_.Printf("# Args: ");
+      logfile_.Write(commandLine_.c_str(), commandLine_.size());
+      logfile_.Printf("\n");
+    }
     DataSetList tops = State_.DSL().GetSetsOfType("*", DataSet::TOPOLOGY);
     if (!tops.empty()) {
       logfile_.Printf("# Loaded topologies:\n");
```

**Comment by drroe** [comment]
Fixed by #959
